cmd/translate/root.go

```diff
@@ -0,0 +1,59 @@
+package translate
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands.
+var rootCmd = &cobra.Command{
+	Use:   "translate",
+	Short: "Enables translation for Cloud-native systems",
+	Long:  `Enables translation for Cloud-native systems`,
+	// Uncomment the following line if your bare application
+	// has an action associated with it:
+	// Run: func(cmd *cobra.Command, args []string) { },
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	err := rootCmd.Execute()
+	if err != nil {
+		os.Exit(1)
+	}
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+	rootCmd.AddCommand(translateCmd)
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is translate/translate.yml)")
+}
+
+// initConfig reads in config file and ENV variables if set.
+func initConfig() {
+	if cfgFile != "" {
+		// Use config file from the flag.
+		viper.SetConfigFile(cfgFile)
+	} else {
+		// Find current dir.
+		dir, err := os.Getwd()
+		cobra.CheckErr(err)
+
+		// Search config in current directory with name ".translate" (without extension).
+		viper.AddConfigPath(dir)
```

Comment:
I am not sure if it is needed, because it works just fine if you remove it, as it in the same dir as main.go
```go
	if cfgFile != "" {
		// Use config file from the flag.
		viper.SetConfigFile(cfgFile)
	} else {
		// Sets default config.
		viper.SetConfigFile("translate.yml")
	}
```